### PR TITLE
Remove unnecessary constraints on `ExternalPtr<T>`

### PR DESCRIPTION
**Pull request by CGMossa**

### CODE REVIEW

File: .Rprofile
Comment (JosiahParry) [inline .Rprofile:1]: delete

File: Untitled.R
Comment (JosiahParry) [inline Untitled.R:1]: delete

**Comment by CGMossa** [comment]
@JosiahParry Sorry! the git-calamity was rectified.

**Comment by JosiahParry** [comment]
Do you mind syncing with main so CI passes?

**Review by JosiahParry — APPROVED** [review]
Thanks for updating CI ! This looks great now

**Pull request merged**